tests/std/tests/P0645R10_text_formatting_custom_formatting/test.cpp

```diff
@@ -49,8 +71,78 @@ constexpr void test_disabled_formatter_is_disabled() {
     static_assert(!is_move_assignable_v<F>);
 }
 
+template <class T, class charT>
+void test_custom_equiv_with_format(const charT* fmt_string, T&& val) {
+    assert(format(fmt_string, custom_formattable_type<T>{forward<T>(val)}) == format(fmt_string, val));

```

Comment:
`forward` seems like a bad idea when we use `val` twice - why not use an lvalue in both cases? (Also on 81.)